angularjs-portal-home/src/main/webapp/my-app/search/partials/search-results.html

```diff
@@ -1,144 +1,250 @@
-  </div>
-</div>
+    <!-- MyUW RESULTS ONLY -->
+    <md-tab ng-click="filterTo('myuw')">
+      <md-tab-label>
+        MyUW&nbsp;&nbsp;<span class="badge">{{ myuwFilteredResults.length }}</span>
+      </md-tab-label>
+      <md-tab-body>
+        <md-content layout-padding>
+          <!-- MyUW results -->
+          <div class='search-results-container'>
+            <h4 class="header">MyUW</h4>
+            <hr>
+            <loading-gif data-object='myuwResults'></loading-gif>
+            <div ng-show="myuwResults.length != 0 && myuwFilteredResults.length == 0" class='no-result'>
+              No MyUW results. <a href="apps">Try browsing instead?</a>
+            </div>
+            <div ng-repeat="portlet in myuwFilteredResults = (myuwResults | filter:searchTermFilter | showApplicable:showAll | orderBy:sortParameter | limitTo:searchResultLimit)"
+                 class="result">
+              <h4><a ng-href="{{getLaunchURL(portlet)}}" target="{{::portlet.target}}">{{ portlet.title }}</a> <small ng-if='GuestMode && !portlet.canAdd'>(login to use)</small></h4>
+              <p>{{ portlet.description }}</p>
+              <p>
+                <md-button ng-click="addToHome(portlet)"
+                           ng-if="portlet.canAdd && !portlet.hasInLayout && !GuestMode"
+                           class="md-primary add">
+                  <i class="fa fa-plus"></i> Add to home
+                </md-button>
+                <span ng-if="portlet.canAdd && portlet.hasInLayout && !GuestMode" class="added"><i class="fa fa-check"></i> Added to home</span>
+                <md-button class="md-default" title='See more about {{portlet.title}}' ng-click='navToDetails(portlet, "Search")'>Details</md-button>
+                <span><rating ng-model="portlet.rating" readonly="true" class="rating"></rating></span>
+              </p>
+            </div>
+            <marketplace-load-more ng-show="myuwFilteredResults.length > searchResultLimit"></marketplace-load-more>
+          </div>
+        </md-content>
+      </md-tab-body>
+    </md-tab>
+
+    <!-- DIRECTORY RESULTS ONLY -->
+    <md-tab ng-show="directoryEnabled" ng-click="filterTo('directory')">
+      <md-tab-label>
+        Directory&nbsp;&nbsp;<span class="badge">{{ wiscDirectoryTooManyResults ? '25+' : wiscDirectoryResultCount }}</span>
+      </md-tab-label>
+      <md-tab-body>
+        <md-content layout-padding>
+          <!-- Wisc directory results-->
+          <div ng-show="directoryEnabled" class="search-results-container">
+            <h4 class='header'>Directory</h4>
+            <hr>
+            <loading-gif data-object='wiscDirectoryResults' data-empty="wiscDirectoryResultsEmpty"></loading-gif>
+            <div ng-show="wiscDirectoryResultsEmpty && !wiscDirectoryTooManyResults" class='no-result'>
+              No directory results.
+            </div>
+            <div ng-repeat="item in wiscDirectoryResults | limitTo:wiscDirectoryResultLimit" class="result">
```

Comment:
I made a note about that in the PR. I was thinking maybe that works deserves its own story. I don't mind doing it, since I have plenty of time left in this sprint, but I made the PR in its current state because it's the minimum amount of work that still meets the requirements.